lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala

```diff
@@ -119,6 +120,31 @@ object SnapshotLsifCommand {
           .setSymbol(symbol)
           .build()
         doc.addOccurrences(occ)
+
+        if (isDefinition) {
+          val hover =
+            (
+              for {
+                resultSetId <- lsif.next.get(o.getId)
+                hoverId <- lsif.hoverEdges.get(resultSetId)
+                hover <- lsif.hoverVertexes.get(hoverId)
+              } yield hover
+            ).get
+              .getContentsList
+              .asScala
+              .find { h =>
+                h.getLanguage != Language.UNKNOWN_LANGUAGE.toString.toLowerCase
+              }
+              .get
+              .getValue
```

Comment:
I think this refactoring makes it a bit easier to read and it handles the case when you have multiple hovers

```suggestion
          val hover =
            (
              for {
                resultSetId <- lsif.next.get(o.getId).toList
                hoverId <- lsif.hoverEdges.get(resultSetId).toList
                hover <- lsif.hoverVertexes.get(hoverId).toList
                contents <- hover.getContentsList().asScala
                if contents.getLanguage !=
                  Language.UNKNOWN_LANGUAGE.toString.toLowerCase
              } yield contents.getValue()
            ).mkString("\n")
```